packages/dds/tree/src/test/shared-tree/fuzz/topLevel.fuzz.spec.ts

```diff
@@ -92,6 +98,7 @@ describe("Fuzz - Top-Level", () => {
 			},
 			reconnectProbability: 0.1,
 			idCompressorFactory: deterministicIdCompressorFactory(0xdeadbeef),
+			skipMinimization: true,
```

Comment:
I'm guessing this was left in here by mistake?

Reply:
Whoops, it was left in there accidentally during debugging. Removed `skipMinimization: true`